openstack_dashboard/static/app/core/openstack-service-api/nova.service.js

```diff
@@ -262,6 +270,158 @@
     }
 
     /**
+     * @name startServer
+     * @description
+     * Start a single server by ID.
+     *
+     * @param {String} serverId
+     * Server to start
+     * @returns {Object} The result of the API call
+     */
+    function startServer(serverId, suppressError) {
```

Comment:
Can these be condensed down to a single function? Just pass in operation name and (perhaps) human readable operation name for failure message?